src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -61,19 +111,12 @@ def get(self, request: Request, project) -> Response:
         if not filepath:
             return Response({"detail": "Filepath is required"}, status=400)
 
-        commit_id = request.GET.get("commitId")
-        platform = request.GET.get("platform")
-        sdk_name = request.GET.get("sdkName")
-        abs_path = request.GET.get("absPath")
-        module = request.GET.get("module")
-        package = request.GET.get("package")
-        frame = {}
-        if abs_path:
-            frame["abs_path"] = abs_path
-        if module:
-            frame["module"] = module
-        if package:
-            frame["package"] = package
+        ctx = {
+            "commit_id": request.GET.get("commitId"),
+            "platform": request.GET.get("platform"),
+            "sdk_name": request.GET.get("sdkName"),
+        }
+        mobile_frame = generate_mobile_frame(request.GET)
```

Comment:
Using `mobile_frame` later in the code to decide if this is a mobile platform does not make sense. Either way, here we are.